### PR TITLE
Fix/Update documentation

### PR DESCRIPTION
**Pull request by fuentesj11**

### CODE REVIEW

File: documentation/extensions/dataforms.md [hunk 1]
```diff
@@ -106,22 +112,30 @@ the message to send back.
 **Examples**
 
 In this example we can see how to retrieve a form to fill out, complete the
-form and send it back:
+form, and send it back:
```
Comment (vanitasvitae) [inline documentation/extensions/dataforms.md]: is this comma necessary?

Comment (fuentesj11) [reply documentation/extensions/dataforms.md]: I thought the documentation was describing three separate items in succession.  I am no English expert by any stretch but from what I recall I thought we needed the commas to separate items in the list.  I can remove it, if you would like.

Comment (vanitasvitae) [reply documentation/extensions/dataforms.md]: I'm not a native speaker, so I can only deduce from German, where the comma is unnecessary :D
I'm fine with your change though :)

File: smack-core/src/main/java/org/jivesoftware/smack/XMPPConnection.java [hunk 2]
```diff
@@ -45,7 +45,6 @@
  * // Start a new conversation with John Doe and send him a message.
  * ChatManager chatManager = ChatManager.getInstanceFor(con);
  * chatManager.addIncomingListener(new IncomingChatMessageListener() {
- *     @Override
```
Comment (vanitasvitae) [inline smack-core/src/main/java/org/jivesoftware/smack/XMPPConnection.java]: Please do not mix documentation changes with code changes. Rather create a new PR for changes unrelated to documentation.

Comment (fuentesj11) [reply smack-core/src/main/java/org/jivesoftware/smack/XMPPConnection.java]: Anything touching a *.java (source code) file is considered a code change?

**Comment by vanitasvitae** [comment]
Why did you close the PR?

**Comment by fuentesj11** [comment]
It required less work to just move the modified files to a temp directory, create two new branches and then move the files into one of the two appropriate branches.  I had javadocs changed on 3 source files sprinkled in a few commits.  I had reset the HEAD but a couple of the documentation files were not showing up as changed when running git status.  It was a mess and I am not that experienced with the "advanced" git procedures.  I apologize.

**Comment by vanitasvitae** [comment]
Ah :D don't worry :)

**Pull request closed without merging**